### PR TITLE
[docs] spelling and general improvements (part 4)

### PR DESCRIPTION
**Pull request by daniel-lang**
Part 4 of hopefully 5

Related to #977

### CODE REVIEW

File: docs/decorator-reference.md [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
-* [Subscriber and listener decorators](#subscriber-and-listener-decorators)
-    * [`@AfterLoad`](#afterload)
-    * [`@BeforeInsert`](#beforeinsert)
-    * [`@AfterInsert`](#afterinsert)
-    * [`@BeforeUpdate`](#beforeupdate)
-    * [`@AfterUpdate`](#afterupdate)
-    * [`@BeforeRemove`](#beforeremove)
-    * [`@AfterRemove`](#afterremove)
-    * [`@EventSubscriber`](#eventsubscriber)
+* [Subscriber and listener decorators](./listeners-and-subscribers.md)
```
Comment (pleerock) [inline docs/decorator-reference.md]: I know its a duplication, but I really wanted to keep all decorators on a single page

Comment (daniel-lang) [reply docs/decorator-reference.md]: Okay, I will revert this change

**Comment by daniel-lang** [comment]
I don't know if the links in decorator-reference.md will work. But the old links with .md at the end won't work

**Comment by daniel-lang** [comment]
If you merge [typeorm/typeorm.github.io#17](https://github.com/typeorm/typeorm.github.io/pull/17) all links in these documents will work

**Pull request merged**